src/styles/index.ts

```diff
@@ -4732,6 +4732,11 @@ const styles = (theme: ThemeColors) =>
             borderWidth: 2,
             borderColor: theme.cardBG,
             borderRadius: variables.componentBorderRadiusLarge,
+            height: 200,
+            maxWidth: 400,
```

Comment:
For the RHP, we don't want a maxWidth - it should always be 100% wide right?

Reply:
Hmm I feel like it's better to somehow not need to use a hard-coded value, and just always make sure the receipt as as wide as possible. Can you find a way to not need a number value and do something where it just always grows to the max width available please?

Reply:
I don't know why the font size would impact this at all, that feels odd to me. Something seems wrong there.

Reply:
Works for me.